Symfony/CS/Config/Config.php

```diff
@@ -53,6 +54,13 @@ public function setDir($dir)
         $this->dir = $dir;
     }
 
+    public function setCacheDir($cacheDir)
```

Comment:
@localheinz I'm fine with this being added in 1.6, then having the interface corrected in 2.0. Not up to me though. Ping @keradus.